README.md

```diff
@@ -1,22 +1,336 @@
-  * Traditionally this was solved with HTML. HTML has both data and formatting, not good
-  * Just want pure data, used to be given via XML. XML was annoying
-  * Then came JSON
-  * Show them what JSON looks like. Just a dictionary or array of dictionaries!
-  * JSON is a way of "serializing" data. Remember we need to send everything as strings. Serialization means convert from the ruby format of an object, to a string format of an object. 
-  * Let's write our own JSON post serializer.
-  * Make it a separate class with a single class method that takes in the post object and returns a string of JSON
-  * don't use `to_json` or anything like that. Have them by hand build up a JSON dictionary of the important bits of the post
-  * The show page, make it backed by JSON.
+  1. Understand what JSON is.
+  2. Create and Return simple JSON Objects by hand.
+  3. Load those objects with AJAX into HTML.
+
+## Lesson
+
+Continuing with our blog application, we'll be improving our blog post
+API to provide more structure and more efficient data access. Make sure
+to run `rake db:seed` to set up some starting data!
+
+Last time, we created an endpoint for the single piece of data we wanted
+to access. That worked well enough, but doesn't scale. If we want to use
+AJAX to get all the data from a blog post, we'd be creating so many
+endpoints, and making so many GET requests.
+
+What we really need is one endpoint to make one request and get back
+structured data representing our blog post.
+
+### Serialization
+
+If we're directly accessing a post from our web browser, the answer is
+simple. The controller renders a template that contains HTML and the
+data we need.
+
+We could still do this for an API endpoint, but then that HTML would
+probably get in our way. We want the data, not the formatting. This,
+like so many things, comes back to Separation of Concerns. An API
+endpoint is only concerned with returning the data needed; it's up to
+the API consumer, in this case our AJAX call, to decide how to display
+it.
+
+This is where **serialization** comes in. Serialization is the process
+by which we take "executable" code, in our case a Ruby object, and
+represent it as a string that can be consumed anywhere (remember, the
+Internet is just strings) and then reconstructed back into usable code.
+
+Remember that a Ruby *object* is an instance of a *class*,
+meaning it has been initialized and is running in memory. Serializing an
+object retains the *state*, or current values of all the object's
+attributes, when turning it into a string. This differs from a class
+definition because the class definition tells us what *any* object *could*
+look like, whereas a serialized object tells us what *one* object *does*
+look like. It's an important distinction.
+
+Most of what a modern web API does is gather and serialize objects to be
+passed to the consuming code as a string over HTTP. So how do we do this
```

Comment:
can un "string" this bit as maybe a compromise between the two positions, so we're hitting it but not hammering on it?